packages/@react-types/tooltip/src/index.d.ts

```diff
@@ -17,7 +17,8 @@ import {PositionProps} from '@react-types/overlays';
 export interface TooltipTriggerProps extends PositionProps {
   children: ReactElement[],
   isDisabled?: boolean,
-  onOpenChange?: (isOpen: boolean) => void
+  onOpenChange?: (isOpen: boolean) => void,
+  delay?: boolean
```

Comment:
Why was this prop added?

Reply:
The way I'm reading this, there are actually two options, a tooltip can either appear instantly all the time, or it can have a warmup/cooldown
https://spectrum.adobe.com/page/tooltip/#Immediate-or-delayed-appearance